### PR TITLE
mysql/manual ：Add metrics_alarm and mgt_account topics

### PR DESCRIPTION
**Pull request by molliezhang**
### What type of PR is this?

- Optimiz
- Add Features

### Which issue(s) this PR fixes?
N/A

### What this PR does / why we need it ?
Summary:

1. Add metrics and alarm topics in mysql
2. Add metrics and alarm topics in chronusdb
3. Add mgt_account topics in chronusdb
4. Update more pic in other topics

### Special notes for your reviewer?
N/A

/assign @pethu1110

### CODE REVIEW

File: content/database/mysql/manual/metrics_alarm/set_alarm_rules.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+MySQL Plus 支持对每个缓存节点配置指标告警策略。当业务发生异常时，会触发用户设定的告警阈值，并发送短信或邮件通知用户。
+
+1. 在集群管理页面，点击集群 ID，进入集群详情页面。
+2. 在**告警**页签，选择节点，并点击列表上方**绑定指标告警策略**。
+
+    <img src="../../../_images/alarm.png" alt="告警" style="zoom:50%;" />
+
+3. 在弹出的指标告警策略配置窗口，选择指标告警策略，并点击**提交**。
+
+   > - 若无可选指标告警策略，请参考[创建指标告警策略](#创建指标告警策略)。
```
Comment (pethu1110) [inline content/database/mysql/manual/metrics_alarm/set_alarm_rules.md:59]: image

File: content/database/mysql/manual/metrics_alarm/set_alarm_rules.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    
+   **告警次数** ：当资源持续处于**告警**状态时，连续发送通知的次数。最多为100次。
+   
+   **通知列表** ：选择告警通知列表。
+   
+   <img src="../../../_images/manual_set_alarm_behavior.png" alt="配置告警行为" style="zoom:50%;" />  
+   
+5. 确认配置无误后，点击**提交**，返回指标告警策略配置窗口，即可选择新创建的告警策略。
+
+   > 更多告警策略管理说明，请参见**云监控 CloudSat** [告警服务](../../../../../monitor_service/cloudsat/manual/alarm_service)。
```
Comment (pethu1110) [inline content/database/mysql/manual/metrics_alarm/set_alarm_rules.md:107]: 这里不需要加说明，直接放在步骤5中就可以。

File: content/database/mysql/manual/metrics_alarm/set_alarm_rules.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+   **监控周期** ：可选择`1分钟`和`5分钟`。默认为`5分钟`，免费使用。
+   
+   <img src="../../../_images/manual_set_alarm_basic.png" alt="配置告警行为" style="zoom:50%;" />  
+   
+3. 点击**下一步**，配置告警规则。   
+   
+   点击**添加规则**，并可配置指标规则阈值和告警级别，一个策略可添加多条指标规则。
+
+   > 有多条指标啊规则时，任何一条规则满足条件都会触发告警。
```
Comment (pethu1110) [inline content/database/mysql/manual/metrics_alarm/set_alarm_rules.md:89]: 最好加上“说明”

File: content/database/mysql/manual/metrics_alarm/view_monitor_metrics.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+draft: false
+---
+
+
+
+MySQL Plus 监控告警是通过 QingCloud 云监控告警服务为集群服务器的资源和服务提供监控管理。当集群监控项超过阈值时触发告警，并通过短信、邮件等形式发送告警通知。
+
+QingCLoud 提供的云监控 CloudSat，可对 MySQL Plus 的运行状态进行日常监控。您可以通过 CloudSat 管理控制台，一站式监控和告警 MySQL Plus 各类服务、资源指标。您可以在 MySQL Plus 集群详情页面，查看主实例监控和查看备实例监控。
+
+> **注意**
```
Comment (pethu1110) [inline content/database/mysql/manual/metrics_alarm/view_monitor_metrics.md:16]: 没有分行。
image

File: content/database/mysql/manual/metrics_alarm/view_monitor_metrics.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+> 由于云监控 CloudSat 默认监控扫描周期为5分钟，则当前显示5～10分钟前的集群状态。
+
+## 前提条件
+
+- 已获取 QingCloud 管理工作台登录账号和密码，且已获取集群查看权限。
+- 已创建 MySQL Plus 集群，集群状态为**活跃**，且服务状态为**正常**。 
+  
+  > 更新中、异常、删除状态的集群，无法获取其监控指标。当集群重启或恢复后，即可正常查看。
```
Comment (pethu1110) [inline content/database/mysql/manual/metrics_alarm/view_monitor_metrics.md:24]: 增加“说明”

File: content/database/mysql/manual/metrics_alarm/view_monitor_metrics.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+  
+- 集群服务已正常运行一段时间。
+  
+  > 监控扫描周期默认为5分钟，新增节点暂无法查看监控信息。
```
Comment (pethu1110) [inline content/database/mysql/manual/metrics_alarm/view_monitor_metrics.md:28]: 增加“说明”

File: content/database/mysql/manual/metrics_alarm/set_alarm_rules.md (hunk 1 above)
Comment (molliezhang) [reply content/database/mysql/manual/metrics_alarm/set_alarm_rules.md:59]: remove “ -”

File: content/database/mysql/manual/metrics_alarm/set_alarm_rules.md (hunk 3 above)
Comment (molliezhang) [reply content/database/mysql/manual/metrics_alarm/set_alarm_rules.md:89]: done

**Review by pethu1110 — APPROVED** [review]

**Pull request merged**